client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -0,0 +1,272 @@
+    "Liste de diffusion",
+"Videos":
+    "Vidéos",
+"Wiki":
+    "Documentations",
+"How to Cite Galaxy":
+    "Comment citer Galaxy",
+"Interactive Tours":
+    "Guides interactifs",
+"User":
+    "Utilisateur",
+"Login":
+    "Authentification",
+"Register":
+    "Enregistrement",
+"Logged in as":
+    "Authentifié en tant que",
+"Custom Builds":
+    "Mes génomes Builds de référence",
+"Logout":
+    "Déconnection",
+"Saved Histories":
+    "Historiques sauvegardés",
+"Saved Datasets":
+    "Données sauvegardés",
+"Saved Pages":
+    "Pages sauvegardées",
+// ---------------------------------------------------------------------------- histories
+// ---- history-model
+// ---- history-view
+"This history is empty" :
+    "Cet historique est vide",
+"No matching datasets found" :
+    "Aucunes données correspondantes n'a été trouvées",
+"An error occurred while getting updates from the server" :
+    "Une erreur s'est produite lors de l'envoi des mises à jour depuis le serveur",
+"Please contact a Galaxy administrator if the problem persists" :
+    "Veuillez contacter un administrateur de l'instance Galaxy si ce problème persiste",
+//TODO:
+//"An error was encountered while <% where %>" :
+//false,
+"Search datasets" :
+    "Recherche dans l'ensemble des données",
+"You are currently viewing a deleted history!" :
+    "Vous consultez actuellement un historique supprimé!",
+"You are over your disk quota" :
+    "Vous avez dépassé votre quota disque",
+"Tool execution is on hold until your disk usage drops below your allocated quota" :
+    "L'exécution de l'outil est en attente tant que votre utilisation d'espace disque dépasse le quota attribué",
+"All" :
+    "Tout",
+"None" :
+    "Aucun",
+"For all selected" :
+    "Pour toute la sélection",
+
+// // ---- history-view-edit
+"Edit history tags" :
+    "Editer les mots-clés de l'historique",
+"Edit history Annotation" :
+    "Editer l'annotation de l'historique",
+"Click to rename history" :
+    "Cliquer pour renommer l'historique",
+// multi operations
+"Operations on multiple datasets" :
+    "Opérer sur plusieurs jeux de données en même temps",
+"Hide datasets" :
+    "Cacher les jeux de données",
+"Unhide datasets" :
+    "Restaurer les jeux de données cachés",
+"Delete datasets" :
+    "Supprimer les jeux de données",
+"Undelete datasets" :
+    "Restaurer les jeux de données supprimés",
+"Permanently delete datasets" :
+    "Supprimer définitivement les jeux de données",
+"This will permanently remove the data in your datasets. Are you sure?" :
+    "Cela supprimera de manière permanente les données de votre historique. Êtes-vous certains?",
+
+// // ---- history-view-annotated
+"Dataset" :
+    "Jeu de données",
+"Annotation" :
+    "Annotation",
+
+// ---- history-view-edit-current
+"This history is empty. Click 'Get Data' on the left tool menu to start" :
+    "Cet historique est vide. Cliquer sur 'Téléverser des données' au niveau du menu d'outils à gauche pour démarrer",
```

Comment:
Yeah, this is a sticky one.  Let me see what I can do about "Get Data", since it's so high profile.  We certainly won't be able to do all tools, but maybe this section.